site/content/en/_index.html

```diff
@@ -44,7 +44,17 @@
 {{< /blocks/section >}}
 
 
-{{< blocks/section color="primary"  >}}
+{{< blocks/section color="primary" >}}
+<div class="col">
+		<h2 class="text-center pb-3">Companies using Open Match</h2>
+		<p class="text-center showcase">
+			<a href="http://unity.com/"><img alt="unity" width="150" src="images/unity.png" /></a>
```

Comment:
The image reference here is `images/unity.png`, but you added the logo as the file `images/UnityLogo.png`, so it doesn't show up properly.

Reply:
ouch, will send a new one, it was a last-minute mistake x) thanks!